wp-dependency-installer.php

```diff
@@ -253,7 +273,9 @@ public function admin_init() {
 
 			// Generate admin notices.
 			foreach ( $this->config as $slug => $dependency ) {
-				$is_required = $this->is_required( $dependency );
+				$is_required     = $this->is_required( $dependency );
+				$suggest_active  = ! empty( $dependency['activate_notice'] );
+				$suggest_install = ! empty( $dependency['install_notice'] );
```

Comment:
Third party devs can now use the following filter `wp_dependency_config_{$slug}` in order to display an install notice only if another external specific condition is meet ( eg. show install notice only if another companion plugin is currently installed on your system )

Reply:
I'm not sure I follow the actual use case.

Reply:
I close this due to outdated code